zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/identity/MappingCreateProcessor.java

```diff
@@ -68,6 +71,21 @@ public void processNewCommand(final TypedRecord<MappingRecord> command) {
     }
 
     final var record = command.getValue();
+    if (!StringUtils.hasText(record.getMappingId())
+        || !StringUtils.hasText(record.getName())
+        || !StringUtils.hasText(record.getClaimName())
+        || !StringUtils.hasText(record.getClaimValue())) {
```

Comment:
:x: Let's not depend on Spring just for `StringUtils`. A simple `String#isBlank` should be enough.